src/stores/feedbacks.js

```diff
@@ -18,17 +19,18 @@ export const useFeedbackStore = defineStore('feedbacks', {
   actions: {
     async fetchFeedbacks() {
       this._isLoading = true
-      await getDocs(collection(db, 'feedbacks'))
-        .then(async (querySnapshot) => {
+
+      // TO avoid a subscription every time we call fetch FeedBacks
+      if (!this._unSubscribe)
+        this._unSubscribe = onSnapshot(collection(db, 'feedbacks'), async (querySnapshot) => {
```

Comment:
Feedbacks are not frequent.
Do we really need to subscribe to them?

Reply:
It is easier to handle the reload of the data with on snapshot. 
Use Case
The Admin is connected and loaded some feedback
Then a user submits a new feedback
If the Admin don't reload the page, he will not remarque the new feedback.